storage/flux/table.gen.go

```diff
@@ -86,6 +86,10 @@ func (t *floatTable) Do(f func(flux.ColReader) error) error {
 
 func (t *floatTable) advance() bool {
 	a := t.cur.Next()
+	if t.cur.Err() != nil {
```

Comment:
I wonder if this error check should be behind `a.Len() == 0` - should error'd cursors be returning values?

Reply:
As the code is, the erred cursor is not returning data, correct? Because we don't do the copy from `a`.  Are you suggesting we should do the copy even on error, or we shouldn't check for the error unless `a.Len == 0`?

Reply:
I'm suggesting that instead of checking for errors on every read, we either check:

a) when we see `a.Len() == 0`
b) after calling `t.cur.Close()`, similar to what `cursorToLineProtocol` does

Reply:
I'm hoping @jsternberg might have some insight into the intended usage.

Reply:
I understand. I would prefer to check on every `Next()` call. I prefer to report errors as soon as we can detect them. The caller can ignore them or not as preferred.